### PR TITLE
Project initialization

### PR DESCRIPTION
**Pull request by divinecharlotte**
## IN THIS PR WE IMPLEMENTED THE FOLLOWING
- Created a class MyList that has an instance variable @list.
- In `MyList` implemented a method `#each` that yields successive members of `@list` and uses the MyEnumerable module.
- Created a module `MyEnumerable` that implements the following methods (they have the same functionality as methods in [Enumerable](https://ruby-doc.org/core-3.0.0/Enumerable.html)):
- `all?`
- `any?`
- `filter`

### CODE REVIEW

File: class.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+require_relative 'module'
+class Mylist
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+end
```
Comment (Danieldotcomcoder) [inline class.rb]: Inside Mylist class you should implement #each method that yields all members of @list in succession as mentioned in the project requirements , Kindly fix this issue 👍

**Review by Danieldotcomcoder — CHANGES_REQUESTED** [review]
Hello @divinecharlotte , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: class.rb (hunk 1 above)
Comment (divinecharlotte) [reply class.rb]: THANKS @Danieldotcomcoder

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @divinecharlotte , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by divinecharlotte** [comment]
THANKS @Danieldotcomcoder

**Pull request merged**